packages/veritone-react-common/src/components/SDOTable/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { arrayOf, object, objectOf, bool } from 'prop-types';
+import { Table, PaginatedTable, Column } from 'components/DataTable';
+import { map, startCase, partial, omit } from 'lodash';
+import DotDotDot from 'react-dotdotdot';
+import { format } from 'date-fns';
+
+export default class SDOTable extends React.Component {
+  static propTypes = {
+    data: arrayOf(object).isRequired,
+    schema: objectOf(object).isRequired,
+    paginate: bool
+  };
+
+  static defaultProps = {
+    paginate: false
+  };
+
+  getRowData = i => {
+    return this.props.data[i];
+  };
+
+  renderCell = (data, dataType) => {
+    let formattedData;
+
+    switch (dataType) {
+      case 'dateTime':
+        formattedData = format(data, 'M/D/YYYY h:mm A');
+        break;
+      case 'geoPoint':
+        formattedData = `[${data}]`;
+        break;
+      default:
+        formattedData = data;
+    }
+
+    return <DotDotDot clamp={2}>{formattedData}</DotDotDot>;
+  };
+
+  render() {
+    const tableProps = omit(this.props, ['data', 'schema', 'paginate']);
+
+    const TableComp = this.props.paginate ? PaginatedTable : Table;
```

Comment:
really minor but would prefer to name this TableComponent, not gaining anything by shortening the variable here.